k8s/base/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ./deploy-server.yaml
+  - ./svc-server.yaml
+  
+configMapGenerator:
```

Comment:
This should go in the overlay in `k8s/envs/local/kustomization.yaml`

Reply:
Same as below?